packages/runtime/container-runtime/src/opLifecycle/remoteMessageProcessor.ts

```diff
@@ -179,7 +179,7 @@ export class RemoteMessageProcessor {
 /** Takes an incoming message and if the contents is a string, JSON.parse's it in place */
 export function ensureContentsDeserialized(
```

Comment:
Consider adding a param-annotation block. 

PS: Curious why some of the methods (even `private` ones) have it and public/exported ones don't!

Reply:
Will do. Maybe too many cooks...?

Reply:
Naah! it's more of loosely followed coding conventions across the code-base.  :smile: